rocketmq-flink/pom.xml

```diff
@@ -34,7 +34,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <rocketmq.version>4.7.1</rocketmq.version>
-        <flink.version>1.13.0</flink.version>
+        <flink.version>1.13.1</flink.version>
```

Comment:
Why rocketmq-flink package has flink jar?That will lead a package conflict when run on Flink.

![image](https://user-images.githubusercontent.com/31175234/129822920-be04ac7f-53c8-4c28-b749-7088237fced2.png)

It should set scope to "provided".

```xml
		<dependency>
			<groupId>org.apache.flink</groupId>
			<artifactId>flink-streaming-java_${scala.binary.version}</artifactId>
			<version>${project.version}</version>
			<scope>provided</scope>
		</dependency>
		<dependency>
			<groupId>org.apache.flink</groupId>
			<artifactId>flink-java</artifactId>
			<version>${project.version}</version>
			<scope>provided</scope>
		</dependency>
```